### PR TITLE
Add to docs: WF model, multiple chromosomes, and hybrid simulations #447

### PR DESCRIPTION
**Pull request by DomNelson**
A starting point for DTWF docs, happy to extend/add sections that might be useful.

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -673,6 +673,113 @@ breakpoints follows the recombination rate closely.
...
+Multiple chromosomes can be simulated by specifying a recombination map with
+hotspots between chromosomes. For example, to simulate two chromosomes each 1
+Morgan in length:
+
+.. code-block:: python
+
+    rho = 1e-8
+    positions = [0, 1e8-1, 1e8, 2e8-1]
+    rates = [rho, 0.5, rho, 0]
+    num_loci = positions[-1]
```
Comment (jeromekelleher) [inline docs/tutorial.rst]: Need to cast this as an int it won't work below.

File: docs/tutorial.rst [hunk 2]
```diff
@@ -673,6 +673,113 @@ breakpoints follows the recombination rate closely.
...
+
+In some situations Wright-Fisher simulations are desireable yet less
+computationally efficient than coalescent simulations, for example simulating a
+small sample in a recently admixed population. In these cases, a hybrid model
+offers an excellent tradeoff between simulation accuracy and performance.
+
+This involves two steps:
+
+1. Simulating a fixed number of Wright-Fisher generations using the
+   ``__tmp_max_time`` parameter and returning a partial tree sequence (which
```
Comment (jeromekelleher) [inline docs/tutorial.rst]: Ugh --- we'll have to fix this to actually work properly and call it ``max_time`` before release.

**Comment by DomNelson** [comment]
Can we merge this?

**Comment by jeromekelleher** [comment]
No, I think we need to fix up the discussion on mixed simulations first. We don't want people using the ``__tmp_max_time`` thing, and merging this will push it to readthedocs.

**Comment by DomNelson** [comment]
@jeromekelleher is this looking mergeable?

**Comment by jeromekelleher** [comment]
Looks great, thanks @DomNelson!

**Pull request merged**